instance/templates/instance/ansible/ephemeral.yml

```diff
@@ -0,0 +1 @@
+EDXAPP_IMPORT_EXPORT_BUCKET: null
```

Comment:
@bradenmacdonald Ah thanks! I didn't know about the `!!` syntax. Just updated the branch.